### DISCUSSION TITLE
Spatie media plugin model setup for multiple, webp, responsive images

### DISCUSSION THREAD

**Discussion by faizananwerali** [filamentphp/filament#11839] [Help]
### Package

Form builder

### Package Version

v3

### How can we help you?

How do I setup my model for this

```php
SpatieMediaLibraryFileUpload::make('project_images')
    ->multiple()
    ->responsiveImages()
```

I need your help to set up my model to use this multiple, webp, and responsive images in webp also.

Can any of you guys help me with a sample model code to use this code?

Basically I want to create multiple projects with each project having multiple images in webp format, each image also having responsive images which should also be webp format

**Comment by eduardonwa** [comment]
Hello @faizananwerali did you get an answer on this issue?

i am currently trying to figure out why my images don't display on my live server, i`m trying to see if by making the generated images webp solves the issue. however, on my dev environment i can see the images just fine but still jpg is the format that the generated images convert to.

Cheers.
